src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const carRent = 40;
+  const longRent = days * carRent - 20;
+  const weekRent = days * carRent - 50;
```

Comment:
`days * carRent` it is repeated - use variable `default price`
and 2 variable for discounts